transcom/mymove
# create office user available list for safety moves

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A981461&RoomContext=TeamRoom%3A848240&concept=TeamRoom)

## Summary

This story will cover the SC, TOO, and TIO office users.  Supervisors with safety privilege should see safety moves and their assignment drop down should only include users with safety privileges.

### How to test

1. You will need several SC accounts setup, all in the same transportation office.
   a. SC supervisor with safety privilege.
   b. SC user (non-supervisor) with safety privilege.
   c. SC supervisor without safety privilege.
   d. SC user (non-supervisor) without safety privilege.
2. Setup at least one non-safety move to have the counseling office match your SC user's transportation office.
3. Setup at least one safety move to have the counseling office match your SC user's transportation office.
4. Login as the SC supervisor with safety priv and:
   a. Verify that you can see only SC safety priv office members on the safety move you made in step 3 (that is assigned to your counseling office).
   b. Verify that you see all SCs setup above on a non-safety move in your counseling office.
5. Login as the SC supervisor without safety priv and since you won't see the safety move you created, just verify that you can still see all SC members you expect in the dropdown for moves in your counseling office.
6. Now go back to step 1 and create or change your users to TOO and run through steps 2-5.
7. Finally, go back to step 1 and create or change your users to TIO and run through steps 2-5.
